.github/workflows/rpm_build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           dnf install -y rpm-build rpmdevtools git make
           dnf module -y install go-toolset 
           rpmdev-setuptree
-          echo $GITHUB_SHA | cut -c1-8 > .commit
+          make .version
```

Comment:
Does this work? From what I remember from working on this a while back, git commands don't work here since it's not a git repository. That's why it's using the environment variable.